### ISSUE TITLE
MDCSnackbar might become inaccessible to VoiceOver.

### ISSUE THREAD

**Issue by sergiocoll** [material-components/material-components-ios#8496]
We present a Snackbar while dismissing a VC that would allow users to Undo the deletion of a Bookmark. In some cases the Snackbar is not accessible to Voiceover, even though we can't reproduce constantly. This seems to be happening on Plus/Max devices only.

The expectation would be that the Snackbar is always selectable by Voiceover. Here's a link to a video: https://drive.google.com/open?id=1O8bf0bJsztHf9lO2ghGHtAJKmv8diXCN

Please take a look at this public Chromium bug for more information:
https://bugs.chromium.org/p/chromium/issues/detail?id=985762
